WordPress/Classes/Services/MediaService.m

```diff
@@ -565,4 +567,15 @@ - (RemoteMedia *)remoteMediaFromMedia:(Media *)media fieldsToUpdate:(NSArray<NSS
     return remoteMedia;
 }
 
+- (NSMutableDictionary *)blogPropertiesToTrack:(Blog *)blog {
+    NSMutableDictionary *properties = [NSMutableDictionary dictionary];
+    BOOL isJetpackConnected = blog.jetpack != nil && blog.jetpack.isConnected;
```

Comment:
I may be wrong here @SiobhyB but I was thinking Objective-C will return `nil` even if `blog.jetpack` is `nil`, e.g.:

`BOOL isJetpackConnected = blog.jetpack.isConnected != nil;`

But it'd be good to double-check!

Reply:
@twstokes, thanks for this note! I got the following error when testing the suggested line: `Comparison between pointer and integer ('BOOL' (aka '_Bool') and 'void *')`. I think because `isConnected` is a BOOL that can't be compared to `nil`.

To get the line more streamlined though, prompted by your comment, I've made an update in https://github.com/wordpress-mobile/WordPress-iOS/pull/22360/commits/7cb3d7f1fe7198ac0bb8dba06902dc1ad8f7cea4 to:

`BOOL isJetpackConnected = blog.jetpack.isConnected ? YES : NO;`

As I understand, if `blog.jetpack` is `nil`, the above will still safely return `NO`. Let me know if that seems right to you!

Reply:
Ah! Thanks for double-checking - I'm rusty on Objective-C. 😄

In that case, would `properties[WPAppAnalyticsKeyIsJetpack] = @(blog.jetpack.isConnected);` work? If `jetpack` is `nil` shouldn't Objective-C evaluate it to falsy / boolean `NO` like it did with your latest change?

Reply:
Ah, yes, you're right! Thank you. I wasn't seeing the forest for the trees there, I updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/22360/commits/12850499ab9f93ff01d72c43a7d723b6115d9ea0.

Reply:
Awesome! I didn't see that solution either initially, but I'm glad we arrived to it!